packages/mdc-form-field/_mixins.scss

```diff
@@ -47,23 +47,60 @@
     // stylelint-disable-next-line selector-max-type
     > label {
       @include feature-targeting-mixins.targets($feat-structure) {
-        @include rtl-mixins.reflexive-property(margin, 0, auto);
         @include rtl-mixins.reflexive-property(padding, variables.$item-spacing, 0);
 
         order: 0;
       }
     }
+
+    &:not(.mdc-form-field--space-between) {
+      > label {
+        @include feature-targeting-mixins.targets($feat-structure) {
+          @include rtl-mixins.reflexive-property(margin, 0, auto);
+        }
+      }
+    }
+
+    &.mdc-form-field--space-between {
+      > label {
+        @include feature-targeting-mixins.targets($feat-structure) {
+          @include rtl-mixins.reflexive-property(margin, auto, 0);
+        }
+      }
+    }
```

Comment:
The purpose of the margin is to prevent the label from moving when the form field's width increases. We only add the margin when the field is not space-between and it's not needed with space-between, so this section can be removed.

```suggestion
```

Reply:
done :)